xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -1062,8 +1066,7 @@
       <Docs>
         <typeparam name="T">The type of reference.</typeparam>
         <param name="source">The reference to subtract the offset from.</param>
-        <param name="bytesOffset">The offset to subtract.</param>
-        <param name="byteOffset">To be added.</param>
+        <param name="byteOffset">The offset to subtract.</param>
```

Comment:
You removed one of the parameters from here. This happens when the parameter name does not match in the source file and the ref file. The official one is the one in the ref file. Which one is the correct one, can you please confirm?

Reply:
yes byteOffset is the name of parameter in the ref file

Reply:
That is interesting because the Parameters section, signatures, etc. only shows one of them. 

@joelmartinez is this a bug in mdoc?

Reply:
I remembered there's another possible case: if the parameter name is changed from one version to another, they both show up, and both need to be documented, not just one.

@joelmartinez may know the true answer for this.

Reply:
This issue should be resolved in the next mdoc release (5.7.4.10) ... ETA is soon, but not immediately, so manual interventions for this is ideal.

Reply:
I thought of this too but then the parameter would have additional information to convey this like here where you see different frameworkalternate values for each parameter:
https://github.com/dotnet/dotnet-api-docs/blob/master/xml/Microsoft.VisualBasic/Collection.xml#L1161